pkg/pipelines/tekton/tasks.go

```diff
@@ -5,7 +5,24 @@ import (
 	"strings"
 )
 
-var DeployerImage = "ghcr.io/knative/func-utils:latest"
+var (
+	FuncUtilImage = "ghcr.io/knative/func-utils:latest"
+	DeployerImage string
+	ScaffoldImage string
+	S2IImage      string
```

Comment:
Expected to be set through `ldflags`?

Reply:
yep they could be overriden via ldflags if neccessary, if unset the init block below should fall back to the FuncUtilImage.